mina-daemon/examples/helmfile_seed_node.yaml

```diff
@@ -77,6 +76,16 @@ releases:
           service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: '*'
           service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
           service.beta.kubernetes.io/aws-load-balancer-type: nlb
+        graphql:
+          type: LoadBalancer
+          loadBalancerClass: service.k8s.aws/nlb
+          annotations:
+            external-dns.alpha.kubernetes.io/hostname: seed0-berkeley.minaprotocol.network
+            service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: instance
+            service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: '*'
+            service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+            service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

Comment:
The indentation here seems off. Are you sure this is correct and it renders? There is a service definition and then graphql name with another definition with the same hostname.

Reply:
you're right, I have to change the name because it will collide. I spotted it earlier but later forgot to change it. Good catch